src/util.h

```diff
@@ -195,6 +195,38 @@ void DumpJavaScriptBacktrace(FILE* fp);
 #define UNREACHABLE(...)                                                      \
   ERROR_AND_ABORT("Unreachable code reached" __VA_OPT__(": ") __VA_ARGS__)
 
+#define VALIDATE_FD(env, input)                                                \
```

Comment:
This can be a regular method that returns a `v8::Maybe` (`v8::Nothing` if it throws). The caller can return early when it's nothing which is a general convention in V8/Node.js. It also makes it easier to edit/navigate around this helper.

Reply:
Updated. Thank you!